public/pages/AnomalyCharts/utils/__tests__/anomalyChartUtils.test.ts

```diff
@@ -75,3 +81,81 @@ describe('anomalyChartUtils', () => {
     });
   });
 });
+
+describe('anomalyChartUtils function tests', () => {
+  const alertResponse = {
+    response: {
+      alerts: [
+        {
+          id: 'eQURa3gBKo1jAh6qUo49',
+          version: 300,
+          monitor_id: 'awUMa3gBKo1jAh6qu47E',
+          schema_version: 2,
+          monitor_version: 2,
+          monitor_name: 'Example_monitor_name',
+          monitor_user: {
+            name: 'admin',
+            backend_roles: ['admin'],
+            roles: ['all_access', 'own_index'],
+            custom_attribute_names: [],
+            user_requested_tenant: null,
+          },
+          trigger_id: 'bQUQa3gBKo1jAh6qnY6G',
+          trigger_name: 'Example_trigger_name',
+          state: 'ACTIVE',
+          error_message: null,
+          alert_history: [
+            {
+              timestamp: 1617314504873,
+              message: 'Example error emssage',
+            },
+            {
+              timestamp: 1617312543925,
+              message: 'Example error message',
+            },
+          ],
+          severity: 1,
+          action_execution_results: [
+            {
+              action_id: 'bgUQa3gBKo1jAh6qnY6G',
+              last_execution_time: 1617317979908,
+              throttled_count: 0,
+            },
+          ],
+          start_time: 1616704000492,
+          last_notification_time: 1617317979908,
+          end_time: null,
+          acknowledged_time: null,
+        },
+      ],
+      totalAlerts: 1,
+    },
+  };
+  //TODO: change acknowledgedTime and endTime type to allow null values potentially
```

Comment:
can this TODO be removed?

Reply:
Yes, I will actually change the type on `MonitorAlert` to allow of these because they are values that occur in runtime